internal/log.go

```diff
@@ -11,3 +11,19 @@ func LogVerbose(message string) {
 		fmt.Println(message)
 	}
 }
+
+var LogTraffic bool
+
+type TrafficLogger struct{}
+
+func (l *TrafficLogger) Opened() {}
```

Comment:
Remove the object, and test if it works with a value receiver instead of a pointer